synth.py

```diff
@@ -60,23 +63,77 @@
     library = gapic.py_library(
         service="firestore_admin",
         version=version,
-        bazel_target=f"//google/firestore/admin/{version}:firestore-admin-{version}-py",
-        include_protos=True,
+        # bazel_target=f"//google/firestore/admin/{version}:firestore-admin-{version}-py",
+        # include_protos=True,
+        proto_path=f"google/firestore/admin/{version}",
     )
-    s.move(library / f"google/cloud/firestore_admin_{version}")
+    s.move(library / f"google/firestore/admin_{version}", f"google/cloud/firestore_admin_{version}")
     s.move(library / "tests")
+    s.move(library / "scripts/fixup_keywords.py", f"scripts/fixup_keywords_admin_{version}.py" )
 
     s.replace(
-        f"google/cloud/firestore_admin_{version}/gapic/firestore_admin_client.py",
-        "'google-cloud-firestore-admin'",
-        "'google-cloud-firestore'",
+        f"google/cloud/**/*.py",
+        f"google.firestore.admin_v1",
+        f"google.cloud.firestore_admin_v1",
     )
+    s.replace(
+        f"tests/unit/gapic/**/*.py",
+        f"google.firestore.admin_v1",
+        f"google.cloud.firestore_admin_v1",
+    )
+    s.replace(
+        f"google/cloud/firestore_admin_v1/services/firestore_admin/client.py",
+        f"from google.api_core import operation as ga_operation",
+        f"from google.api_core import operation as ga_operation\nfrom google.api_core import operation",
+    )
+
+
+# ----------------------------------------------------------------------------
+# Edit paths to firestore remove after resolving 
+# https://github.com/googleapis/gapic-generator-python/issues/471
+# ----------------------------------------------------------------------------
+s.replace(
+    f"tests/unit/gapic/**/*.py",
+    f"google.firestore",
+    f"google.cloud.firestore",
+)
+s.replace(
+    f"google/cloud/**/*.py",
+    f"google-firestore-admin",
+    f"google-cloud-firestore",
+)
+s.replace(
+    f"google/cloud/**/*.py",
+    f"google-firestore",
+    f"google-cloud-firestore",
+)
+# TODO(https://github.com/googleapis/gapic-generator-python/issues/471)
+s.replace(
+    f"google/cloud/**/*.py",
+    f"from google.firestore",
+    f"from google.cloud.firestore",
+)
+s.replace(
+    f"docs/**/*.rst",
+    f"google.firestore",
+    f"google.cloud.firestore",
+)
+
 
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(unit_cov_level=97, cov_level=99)
```

Comment:
I'm curious: where is coverage defined now? I wasn't able to find through searching.

Reply:
I see, its defined in `.coveragerc`